watertap/property_models/ion_DSPMDE_prop_pack.py

```diff
@@ -1083,7 +1083,7 @@ def _pressure_osm_phase(self):
         self.pressure_osm_phase = Var(
             self.params.phase_list,
             initialize=1e6,
-            bounds=(5e2, 5e7),
+            bounds=(0, None),
```

Comment:
Does this variable need to have an explicitly defined domain (e.g. `NonNegativeReals`)?

Reply:
@arao53 I think that bounds of (0, None) is equivalent to `NonNegativeReals`